src/codegen/utilities/attribute_helpers.py

```diff
@@ -96,11 +117,92 @@
         "new_name": "ci_velocity_encoder_a_input_term_cfg",
         "deprecated_in": "0.6.6",
     },
+    "ai_meas_types": {
+        "new_name": "ai_supported_meas_types",
+        "deprecated_in": "0.6.6",
+    },
+    "ao_output_types": {
+        "new_name": "ao_supported_output_types",
+        "deprecated_in": "0.6.6",
+    },
+    "ci_meas_types": {
+        "new_name": "ci_supported_meas_types",
+        "deprecated_in": "0.6.6",
+    },
+    "co_output_types": {
+        "new_name": "co_supported_output_types",
+        "deprecated_in": "0.6.6",
+    },
+    "teds_hwteds_supported": {
+        "new_name": "tedshwteds_supported",
+        "deprecated_in": "0.6.6",
+    },
+    "chassis_module_devices": {
+        "new_name": "chassis_module_dev_names",
+        "deprecated_in": "0.6.6",
+    },
+    "compact_daq_chassis_device": {
+        "new_name": "compact_daq_chassis_dev_name",
+        "deprecated_in": "0.6.6",
+    },
+    "compact_rio_chassis_device": {
+        "new_name": "compact_rio_chassis_dev_name",
+        "deprecated_in": "0.6.6",
+    },
+    "field_daq_bank_devices": {
+        "new_name": "field_daq_bank_dev_names",
+        "deprecated_in": "0.6.6",
+    },
+    "field_daq_device": {
+        "new_name": "field_daq_dev_name",
+        "deprecated_in": "0.6.6",
+    },
+    "dev_is_simulated": {
+        "new_name": "is_simulated",
+        "deprecated_in": "0.6.6",
+    },
+    "dev_serial_num": {
+        "new_name": "serial_num",
+        "deprecated_in": "0.6.6",
+    },
+    "tedshwteds_supported": {
+        "new_name": "teds_hwteds_supported",
+        "deprecated_in": "0.6.6",
+    },
```

Comment:
@zhindes Should the new name be just `hwteds_supported`?

Reply:
Yep, let's do that.

Reply:
Changed `teds_hwteds_supported` to `hwteds_supported`.